Mathlib/MeasureTheory/SetSemiring.lean

```diff
@@ -7,8 +7,9 @@ import Mathlib.Data.Nat.Lattice
 import Mathlib.Data.Set.Accumulate
 import Mathlib.Data.Set.Pairwise.Lattice
 import Mathlib.MeasureTheory.PiSystem
+import Mathlib.MeasureTheory.MeasurableSpace.Pi
 
-/-! # Semirings and rings of sets
+/-! # Semirings of sets
```

Comment:
Why this change and the changes below? This file also defines rings of sets, if I'm not mistaken.